src/auth/src/auth.rs

```diff
@@ -0,0 +1,49 @@
+use crate::error::AuthorizationError;
+
+use async_trait::async_trait;
+use fluvio_service::IdentityContext;
+use fluvio_stream_model::core::Spec;
+use serde::{Serialize, Deserialize};
+
+
+#[derive(Eq, PartialEq, Hash, Serialize, Deserialize, Clone, Debug)]
+pub enum Role {
```

Comment:
don't need hard wired role now

Reply:
why would we not need a role if we're going to write the logic to enforce the policy in the fluvio code? @nacardin I thought we all agreed that this was set?

Reply:
That's what I thought... If not where would the role definitions come from? Config?